packages/codegen/templates/app00/pages/_app.tsx

```diff
@@ -0,0 +1,133 @@
+import { CacheProvider } from '@emotion/core'
+import { ConfigProvider } from 'antd'
+import enUS from 'antd/lib/locale/en_US'
+import ruRU from 'antd/lib/locale/ru_RU'
+import dayjs from 'dayjs'
+import { cache } from 'emotion'
+import { gql } from 'graphql-tag'
+import get from 'lodash/get'
+import getConfig from 'next/config'
+import Head from 'next/head'
+import React, { useContext, useEffect } from 'react'
+
+import { withApollo } from '@open-condo/next/apollo'
+import { withAuth } from '@open-condo/next/auth'
+import { LocaleContext, useIntl, withIntl } from '@open-condo/next/intl'
+
+import GlobalStyle from '@condo/domains/common/components/containers/GlobalStyle'
```

Comment:
Should we use `@condo` here?

Reply:
I think we should have the same overridden rules for styles in all apps